src/system/Zikula/Module/SettingsModule/SettingsModuleInstaller.php

```diff
@@ -74,7 +74,12 @@ public function install()
         System::setVar('shorturls', false);
         System::setVar('shorturlstype', '0');
         System::setVar('shorturlsseparator', '-');
-        System::setVar('shorturlsstripentrypoint', true);
+
+        if (function_exists('apache_get_modules') && in_array('mod_rewrite', apache_get_modules())) {
+            // Only strip entry point if "mod_rewrite" is available.
+            System::setVar('shorturlsstripentrypoint', true);
+        }
+
```

Comment:
It's a bit more difficult regarding the installer. I will probably take care of it once the Intaller is finished.